src/view/com/util/UserInfoText.tsx

```diff
@@ -29,7 +30,10 @@ export function UserInfoText({
   attr = attr || 'handle'
   failed = failed || 'user'
 
-  const {data: profile, isError} = useProfileQuery({did})
+  const {data: profile, isError} = useProfileQuery({
+    did,
+    staleTime: STALE.INFINITY,
```

Comment:
Let's set this to fifteen minutes so we don't accumulate a huge cache (the profile objects are large enough to care)

Reply:
Yeah for future documentation -- `staleTime` does not drive useQuery cache lifetimes